packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -70,7 +61,7 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   }, []);
 
   return (
-    <EditorContainer bg="black" as="main" isIFrame={isIFrame}>
+    <EditorContainer bg="black" height="25rem" width="43rem" {...rest}>
```

Comment:
Why do we need `height="25rem" width="43rem"` here if we are already passing those values to `maxHeight` and `maxWidth` in`editorBaseStyles`? I'm not seeing any change with it removed.

Reply:
@zippyzow I think so that we can pass in an override by consumers of this component. If we just keep in styled components, it's a little harder

Reply:
I think we want to keep maxWidth, width, and height here, and not in EditorBaseStyles! Happy to jump on a slack thread to clarify.

Reply:
We just want to make sure we can override these easy from portal app or any consumer that needs to change those props. :)